pkg/domain/domain.go

```diff
@@ -3394,7 +3394,7 @@ func (do *Domain) planCacheEvictTrigger() {
 
 // SetupWorkloadBasedLearningWorker sets up all of the workload based learning workers.
 func (do *Domain) SetupWorkloadBasedLearningWorker() {
-	wbLearningHandle := workloadbasedlearning.NewWorkloadBasedLearningHandle()
+	wbLearningHandle := workloadlearning.NewWorkloadLearningHandle(do.sysSessionPool)
```

Comment:
as discussed to short the pkg name as wbl as well?

Reply:
Has been discuss in our group. We will use the workloadlearning uniform